sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -2402,6 +2402,11 @@ SIREPO.app.directive('appHeaderRight', function(appDataService, authState, appSt
                       '<li data-help-link="helpUserManualURL" data-title="User Manual", data-icon="list-alt"></li>',
                       '<li data-help-link="helpUserForumURL" data-title="User Forum", data-icon="globe"></li>',
                       '<li data-help-link="helpVideoURL" data-title="Instructional Video" data-icon="film"></li>',
+                      '<li>',
+                        '<a href="{{ slackURL }}" target="_blank">',
+                          '<span data-ng-class="glyphicon"><img width="14" src="/static/img/slack.png" /></span> Sirepo Slack',
+                        '</a>',
+                      '</li>',
```

Comment:
@mkeilman please review https://github.com/radiasoft/sirepo/pull/4229. The perl code in #4228 can be put in a file and run this way (on the code in #4229):

```sh
perl -pi normalize.pl sirepo-components.js
```

To test a new file, e.g. sirepo.js, you run:

```sh
perl -n normalize.pl sirepo.js
```

This will print errors and the line number in sirepo.js where they occurred. You would then fix up those template errors so that the conversion can proceed automatically. Note the changes that are in #4229, e.g. blank lines and missing commas, because you'll run into those in other js files. Once the program runs without output, you can add that file to this pull request.